### PR TITLE
fix bad format

### PR DESCRIPTION
**Pull request by cwdsuzhou**
Fix https://github.com/kubernetes/website/issues/19485

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -527,7 +523,7 @@ configmap/game-config-5-m67dt67794 created
 <!-- To generate a ConfigMap from literals `special.type=charm` and `special.how=very`,
 you can specify the ConfigMap generator in `kusotmization.yaml` as -->
 要从文字 `special.type=charm` 和 `special.how=very` 生成 ConfigMap，可以在 `kusotmization.yaml` 中将 ConfigMap 生成器指定。
-<!-- ```shell
+```shell
```
Comment (SataQiu) [inline content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md]: 感谢贡献 @cwdsuzhou ！
这部分 shell 可以直接删掉，下方已经有译文版本了！

Comment (cwdsuzhou) [reply content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md]: done, thanks

File: content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 2]
```diff
@@ -73,10 +73,6 @@ For example: -->
 你可以使用 `kubectl create configmap` 从同一目录中的多个文件创建 ConfigMap。
 
 例如：
-
-# Create the local directory
```
Comment (tanjunchen) [inline content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md]: Is this en gone now?

**Comment by xichengliudui** [comment]
need fix conflicting files or close pr.

**Comment by cwdsuzhou** [comment]
> need fix conflicting files or close pr.

rebased

**Comment by xichengliudui** [comment]
why delete annotation? after checking the link, I found no errors
`https://kubernetes.io/zh/docs/tasks/configure-pod-container/configure-pod-configmap/`

File: content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 3]
```diff
@@ -73,18 +73,6 @@ For example: -->
...
-mkdir -p configure-pod-container/configmap/
-
-# Download the sample files into `configure-pod-container/configmap/` directory
-wget https://kubernetes.io/examples/configmap/game.properties -O configure-pod-container/configmap/game.properties
-wget https://kubernetes.io/examples/configmap/ui.properties -O configure-pod-container/configmap/ui.properties
-
-# Create the configmap
-kubectl create configmap game-config --from-file=configure-pod-container/configmap/ -->
-
-```shell
```
Comment (SataQiu) [inline content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md]: @cwdsuzhou  87 行需要保留，否则无法与后边的结尾符配对。现在格式已经错乱了，再调整下吧！
可以在这里预览页面：
https://deploy-preview-19489--kubernetes-io-master-staging.netlify.com/zh/docs/tasks/configure-pod-container/configure-pod-configmap/

**Comment by cwdsuzhou** [comment]
> `https://kubernetes.io/zh/docs/tasks/configure-pod-container/configure-pod-configmap/`

seems fixed

File: content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md (hunk 3 above)
Comment (cwdsuzhou) [reply content/zh/docs/tasks/configure-pod-container/configure-pod-configmap.md]: done, PTAL, seems ok now

**Comment by cwdsuzhou** [comment]
ping @SataQiu  
cloud you approve this PR now?

**Pull request merged**